Tests/Security/Authentication/Provider/OAuthProviderTest.php

```diff
@@ -35,7 +35,7 @@ class OAuthProviderTest extends \PHPUnit\Framework\TestCase
     protected $userProvider;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|OAuthProvider
+     * @var OAuthProvider
```

Comment:
Good catch.

Reply:
I'd like to take credit, but it was PHPStan. :laughing: